cpr/util.cpp

```diff
@@ -237,4 +237,18 @@ bool isTrue(const std::string& s) {
     return temp_string == "true";
 }
 
+time_t sTimestampToT(const std::string& st){
+    if (std::is_same<time_t, unsigned long>::value){
```

Comment:
```suggestion
    if (std::is_integral_v<time_t, unsigned long>){
```

This is the more modern way of doing this. Please update all others as well.

Reply:
Hello COM8.
Thank you for your suggestion.
It seems that `std::is_integral_v` can only pass in one parameter.And it can only determine whether it is of type int,but cannot distinguish between different types of int,Is that so?
Welcome to correct.

Reply:
Oh, sorry I meant `std::is_same_v<time_t, unsigned long>`.

Reply:
Okay, I understand. I'll make the changes right away.